CRM/Core/Payment/PayJunction.php

```diff
@@ -57,6 +57,7 @@ public function __construct($mode, &$paymentProcessor) {
    *   the result in an nice formatted array (or an error object)
    */
   public function doDirectPayment(&$params) {
+    require_once 'PayJunction/pjClasses.php'; // necessary for event registrations 
```

Comment:
I think it's better to put the require_once at the top of the files (before class definition)